airbyte-config/models/src/main/java/io/airbyte/config/helpers/LogHelpers.java

```diff
@@ -24,15 +24,33 @@
 
 package io.airbyte.config.helpers;
 
+import io.airbyte.commons.io.IOs;
 import io.airbyte.config.Configs;
+import io.airbyte.config.Configs.WorkerEnvironment;
 import java.io.File;
+import java.io.IOException;
 import java.nio.file.Path;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class LogHelpers {
 
-  // if you update these values, you must also update log4j2.xml
+  private static final Logger LOGGER = LoggerFactory.getLogger(LogHelpers.class);
```

Comment:
should all the logging related stuff be moved to it's own module? or can it stay in models for now?

Reply:
Can stay for now but should be split eventually.